ietfdata/datatracker.py

```diff
@@ -1070,6 +1104,63 @@ def meeting_types(self) -> Iterator[MeetingType]:
         """
         return self._retrieve_multi("/api/v1/name/meetingtypename/", MeetingType)
 
+# =================================================================================================================================
+#   https://datatracker.ietf.org/api/v1/doc/relateddocument/?source=...      - documents that source draft relates to (references, replaces, etc)
+#   https://datatracker.ietf.org/api/v1/doc/relateddocument/?target=...      - documents that relate to target draft
+    
+    def related_documents_source(self,
+            source_id         : str                        = "38559",
```

Comment:
This should be a `Document` rather than a `str`, and should not have a default value.

Reply:
Type has been changed to Document and default value has been removed